src/XeroPHP/Remote/Request.php

```diff
@@ -79,7 +79,7 @@ public function __construct(Application $app, URL $url, $method = self::METHOD_G
         }
 
         //Default to XML so you get the  xsi:type attribute in the root node.
-        $this->setHeader(self::HEADER_ACCEPT, self::CONTENT_TYPE_XML);
+        $this->setHeader(self::HEADER_ACCEPT, $app->getConfig('default_content_type'));
```

Comment:
I _think_ we should use `$app->getConfigOption('xero', 'default_content_type')` here.

If you have a peek at what `__construct` and `getConfig` do, `default_content_type` won't be available on the top level of `$config`; it's nested under `xero.default_content_type`.

Reply:
Nice spot!